src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -112,7 +119,16 @@ export const SelectInstitutionForm = ({ onSubmit, onClose, suggestedInstitutions
                   getOptionLabel={(option) => getLanguageString(option.labels)}
                   renderOption={(props, option) => (
                     <li {...props} key={option.id}>
-                      {getLanguageString(option.labels)}
+                      <Box>
```

Comment:
Om vi ikke skal bruke noe MUI-Box-funksjonalitet
```suggestion
                      <div>
```